tests/contrib/hooks/test_gcs_hook.py

```diff
@@ -71,10 +71,11 @@ def setUp(self):
             self.gcs_hook = gcs_hook.GoogleCloudStorageHook(
                 google_cloud_storage_conn_id='test')
 
-    def test_storage_client_creation(self):
+    @mock.patch(BASE_STRING.format("GoogleCloudBaseHook._get_credentials"))
+    def test_storage_client_creation(self, get_con_mock):
```

Comment:
```suggestion
    def test_storage_client_creation(self, mock_get_credentials):
```

The mock_ * pattern is added to the exclusion list in the pylint. It is never marked as an unused variable. Changing the variable name also makes it easier to understand what is in the variable.